shashi/jan25.js

```diff
@@ -0,0 +1,48 @@
+
+const isPalindrome = str => {
+  let strNew;
+  if (typeof str === "number") {
+    strNew = str.toString();
+  }
+  if (str.length === 1 || str.length === "" || str === " ") {
+    return -1;
+  }
+  strNew = (str.split(" ")).join("").toLowerCase();
+
+  let strLen = Math.floor(strNew.length - 1 / 2);
```

Comment:
what's the difference between `let` and `const`?

Reply:
let is like var and can be reassigned, where as const cannot be reassigned.

Reply:
are you re-assigning strLen?

Reply:
I am not re-assigning strLen.